ospfd/ospfd.c

```diff
@@ -132,6 +132,10 @@ void ospf_router_id_update(struct ospf *ospf)
 			 * !(virtual | ptop) links
 			 */
 			ospf_nbr_self_reset(oi, router_id);
+
+			if (if_is_operative(oi->ifp) && oi->state == ISM_Down
```

Comment:
could we maybe have a comment about what's being checked/done here?

Reply:
fixed